components/console/helpers/questionhelper.rst

```diff
@@ -234,6 +234,32 @@ You can also specify if you want to not trim the answer by setting it directly w
         $name = $helper->ask($input, $output, $question);
     }
 
+Accept Multiline Answers
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the question helper stops reading user input when it receives a newline
+character (i.e., when the user hits ``ENTER`` once). However, you may specify that
+the response to a question should allow multiline answers by passing ``true`` to
+:method:`Symfony\\Component\\Console\\Question\\Question::setMultiline`::
+
+    use Symfony\Component\Console\Question\Question;
+
+    // ...
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        // ...
+        $helper = $this->getHelper('question');
+
+        $question = new Question('How do you solve world peace?');
```

Comment:
very inspirational quote for the day :)

Reply:
I figured it was appropriate for an answer that needs more than one line of input. 😉